docs/source/index.rst

```diff
@@ -1,25 +1,30 @@
-Welcome to the VUALTO Task Engine documentation!
+TASK ENGINE Documentation
 ===========================================
 
 .. toctree::
    :maxdepth: 2
    :hidden:
    
-   All Documentation <https://vualto-public-documentation.readthedocs-hosted.com/en/latest/index.html>
+   ALL DOCUMENTATION <https://docs.vualto.com>
+   TASK ENGINE <https://docs.vualto.com/projects/task-engine/en/latest/index.html>
+
+The Task Engine is a product that facilitates the creation of VOD from online and offline sources. It is designed to allow simple or complicated custom workflows to be developed, deployed and maintained easily and quickly. The Task Engine is exposed via a REST API and reports back to the client system via a callback mechanism.
+
+How it works
```

Comment:
@jonmallVu index.rst is the overview page now remember. Integration and Authentication were moved to https://docs.vualto.com/projects/task-engine/en/te-new-template-changes/DeveloperDocumentation/TaskEngine.html To prevent index from being too cluttered